pages/examples/boilerplate/head.js

```diff
@@ -1,25 +1,60 @@
 window.addEventListener("DOMContentLoaded", function () {
-  var sdkJs = document.createElement("script");
-  sdkJs.src = "../../../dist/kustotrender.js";
+  // Multi-path fallback logic so examples work regardless of nesting or how dist was copied
+  var scriptPaths = [
```

Comment:
what are these changes. Please correct me, the changes are for tests right. Was there a specific reason for changes on the src and not the test alone. These look flaky too referencing dist